dependencies/src/main/kotlin/de/fayard/dependencies/internal/MigrationToDependenciesConstants.kt

```diff
@@ -0,0 +1,98 @@
+package de.fayard.dependencies.internal
+
+import de.fayard.versions.internal.ArtifactVersionKeyReader
+import de.fayard.versions.internal.cli.AnsiColor
+import de.fayard.versions.internal.cli.CliGenericUi
+import de.fayard.versions.internal.getVersionProperties
+import de.fayard.versions.internal.hasHardcodedVersion
+import de.fayard.versions.internal.retrieveVersionKeyReader
+import kotlinx.coroutines.*
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.ExternalDependency
+import kotlin.coroutines.coroutineContext
+
+internal fun Configuration.shouldBeIgnored(): Boolean {
+    return name.startsWith(prefix = "_internal") // Real-life example: _internal_aapt2_binary (introduced by AGP)
+        || name in ignoredConfigurationNames || name.startsWith('-')
+    //TODO: If unwanted configurations still get through, we can filter to known ones here, like
```

Comment:
What about all the cases that we don't know about?
For example, the gradle-versions-plugin add tons of configurations ending with `-copy`
I would rather assume that we don't know all the possible ways that everyone is using Gradle and have a whitelist approach

Reply:
Both blacklists and whitelists are not viable because you'll keep on getting new unexpected configuration names. For example, KMPP allows dynamic naming of configurations.

Regarding the `gradle-versions-plugin`, do you expect people to keep it as they add refreshVersions to their project? Maybe we could advise to replace it, or even detect its presence?